testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeCheckpointsCompatibility.java

```diff
@@ -285,7 +283,13 @@ private void testCheckpointMinMaxStatisticsForRowType(Consumer<String> sqlExecut
 
             // Assert min/max queries can be computed from just metadata
             String explainSelectMax = getOnlyElement(onDelta().executeQuery("EXPLAIN SELECT max(root.entry_one) FROM default." + tableName).column(1));
-            String column = databricksRuntimeVersion.isAtLeast(DATABRICKS_104_RUNTIME_VERSION) ? "root.entry_one" : "root.entry_one AS `entry_one`";
+            String column;
+            if (databricksRuntimeVersion.isPresent()) {
+                column = databricksRuntimeVersion.get().isAtLeast(DATABRICKS_104_RUNTIME_VERSION) ? "root.entry_one" : "root.entry_one AS `entry_one`";
+            }
+            else {
+                column = "root.entry_one";
+            }
```

Comment:
This `else` condition seems redundant. `testCheckpointMinMaxStatisticsForRowType` doesn't run on OSS Delta Lake. 

Same for `testCheckpointNullStatisticsForRowType`.